### ISSUE TITLE
Bitwarden won't run after update from 0.3.1 to 0.3.2

### ISSUE THREAD

**Issue by viperdream26** [hassio-addons/addon-bitwarden#11]
# Problem/Motivation

> Bitwarden refuses to start after update from 0.3.1 to 0.3.2

## Expected behavior

> Expected a clean update and for it to run again.

## Actual behavior

> Won't start, results in errors (see below).

## Logs

```19-11-01 18:51:58 INFO (SyncWorker_18) [hassio.docker.interface] Update image hassioaddons/bitwarden-aarch64:v0.3.1 to hassioaddons/bitwarden-aarch64:0.3.2
19-11-01 18:51:58 INFO (SyncWorker_18) [hassio.docker.interface] Pull image hassioaddons/bitwarden-aarch64 tag 0.3.2.
19-11-01 18:52:13 INFO (SyncWorker_5) [hassio.docker.interface] Update image sabeechen/hassio-google-drive-backup-aarch64:0.98.4 to sabeechen/hassio-google-drive-backup-aarch64:0.99.0
19-11-01 18:52:13 INFO (SyncWorker_5) [hassio.docker.interface] Pull image sabeechen/hassio-google-drive-backup-aarch64 tag 0.99.0.
19-11-01 18:52:18 INFO (SyncWorker_4) [hassio.docker.interface] Update image esphome/esphome-hassio-aarch64:1.13.6 to esphome/esphome-hassio-aarch64:1.14.0
19-11-01 18:52:18 INFO (SyncWorker_4) [hassio.docker.interface] Pull image esphome/esphome-hassio-aarch64 tag 1.14.0.
19-11-01 18:54:02 INFO (SyncWorker_18) [hassio.docker.interface] Stop addon_a0d7b954_bitwarden application
19-11-01 18:54:11 INFO (MainThread) [hassio.homeassistant] Updated Home Assistant API token
19-11-01 18:54:41 INFO (SyncWorker_18) [hassio.docker.interface] Clean addon_a0d7b954_bitwarden application
19-11-01 18:54:41 WARNING (SyncWorker_10) [hassio.docker.interface] Can't grep logs from hassioaddons/bitwarden-aarch64: 409 Client Error: Conflict ("can not get logs from container which is dead or marked for removal")
19-11-01 18:54:42 WARNING (SyncWorker_11) [hassio.docker.interface] Can't grep logs from hassioaddons/bitwarden-aarch64: 409 Client Error: Conflict ("can not get logs from container which is dead or marked for removal")
19-11-01 18:54:44 WARNING (SyncWorker_3) [hassio.docker.interface] Can't grep logs from hassioaddons/bitwarden-aarch64: 409 Client Error: Conflict ("can not get logs from container which is dead or marked for removal")
19-11-01 18:54:46 WARNING (SyncWorker_8) [hassio.docker.interface] Can't grep logs from hassioaddons/bitwarden-aarch64: 409 Client Error: Conflict ("can not get logs from container which is dead or marked for removal")
19-11-01 18:54:48 WARNING (SyncWorker_12) [hassio.docker.interface] Can't grep logs from hassioaddons/bitwarden-aarch64: 409 Client Error: Conflict ("can not get logs from container which is dead or marked for removal")
19-11-01 18:54:50 WARNING (SyncWorker_9) [hassio.docker.interface] Can't grep logs from hassioaddons/bitwarden-aarch64: 409 Client Error: Conflict ("can not get logs from container which is dead or marked for removal")
19-11-01 18:54:52 WARNING (SyncWorker_15) [hassio.docker.interface] Can't grep logs from hassioaddons/bitwarden-aarch64: 409 Client Error: Conflict ("can not get logs from container which is dead or marked for removal")
19-11-01 18:54:54 WARNING (SyncWorker_2) [hassio.docker.interface] Can't grep logs from hassioaddons/bitwarden-aarch64: 409 Client Error: Conflict ("can not get logs from container which is dead or marked for removal")
19-11-01 18:54:57 INFO (SyncWorker_5) [hassio.docker.interface] Stop addon_cebe7a76_hassio_google_drive_backup application
19-11-01 18:55:21 INFO (SyncWorker_5) [hassio.docker.interface] Clean addon_cebe7a76_hassio_google_drive_backup application
19-11-01 18:55:22 WARNING (SyncWorker_6) [hassio.docker.interface] Can't find hassioaddons/bitwarden-aarch64 for cleanup
19-11-01 18:55:23 INFO (MainThread) [hassio.addons] Add-on 'a0d7b954_bitwarden' successfully updated
19-11-01 18:55:27 INFO (SyncWorker_3) [hassio.docker.interface] Cleanup images: ['sabeechen/hassio-google-drive-backup-aarch64:0.98.4']
19-11-01 18:55:44 ERROR (SyncWorker_12) [hassio.docker] Can't start addon_a0d7b954_bitwarden: 403 Client Error: Forbidden ("endpoint with name addon_a0d7b954_bitwarden already exists in network hassio")
19-11-01 18:55:44 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished coro=<AddonManager.update() done, defined at /usr/src/hassio/hassio/addons/__init__.py:177> exception=AddonsError()>
Traceback (most recent call last):
  File "/usr/src/hassio/hassio/addons/__init__.py", line 217, in update
    await addon.start()
  File "/usr/src/hassio/hassio/addons/addon.py", line 484, in start
    raise AddonsError() from None
hassio.exceptions.AddonsError
19-11-01 18:55:53 INFO (MainThread) [hassio.addons] Add-on 'cebe7a76_hassio_google_drive_backup' successfully updated
19-11-01 18:56:31 INFO (SyncWorker_10) [hassio.docker.addon] Start Docker add-on sabeechen/hassio-google-drive-backup-aarch64 with version 0.99.0
19-11-01 18:56:42 INFO (MainThread) [hassio.api.security] /homeassistant/info access from cebe7a76_hassio_google_drive_backup
19-11-01 18:56:42 INFO (MainThread) [hassio.api.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup
19-11-01 18:56:43 INFO (MainThread) [hassio.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup
19-11-01 18:56:43 INFO (MainThread) [hassio.api.security] /snapshots/00499c7e/info access from cebe7a76_hassio_google_drive_backup
19-11-01 18:56:43 INFO (MainThread) [hassio.api.security] /snapshots/af74f702/info access from cebe7a76_hassio_google_drive_backup
19-11-01 18:56:43 INFO (MainThread) [hassio.api.security] /snapshots/a2e45e3e/info access from cebe7a76_hassio_google_drive_backup
19-11-01 18:56:43 INFO (MainThread) [hassio.api.security] /snapshots/711268b7/info access from cebe7a76_hassio_google_drive_backup
19-11-01 19:00:17 INFO (SyncWorker_14) [hassio.docker.interface] Cleanup images: ['esphome/esphome-hassio-aarch64:1.13.6', 'esphome/esphome-hassio-aarch64:latest']
19-11-01 19:00:26 INFO (MainThread) [hassio.addons] Add-on '15ef4d2f_esphome' successfully updated
19-11-01 19:10:16 INFO (SyncWorker_17) [hassio.docker.interface] Clean addon_a0d7b954_bitwarden application
19-11-01 19:10:22 ERROR (SyncWorker_17) [hassio.docker] Can't start addon_a0d7b954_bitwarden: 403 Client Error: Forbidden ("endpoint with name addon_a0d7b954_bitwarden already exists in network hassio")
19-11-01 19:10:35 INFO (SyncWorker_1) [hassio.docker.interface] Clean addon_a0d7b954_bitwarden application
19-11-01 19:10:41 ERROR (SyncWorker_1) [hassio.docker] Can't start addon_a0d7b954_bitwarden: 403 Client Error: Forbidden ("endpoint with name addon_a0d7b954_bitwarden already exists in network hassio")
19-11-01 19:10:41 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished coro=<Addon.start() done, defined at /usr/src/hassio/hassio/addons/addon.py:463> exception=AddonsError()>
Traceback (most recent call last):
  File "/usr/src/hassio/hassio/addons/addon.py", line 484, in start
    raise AddonsError() from None
hassio.exceptions.AddonsError
19-11-01 19:11:02 INFO (SyncWorker_11) [hassio.docker.interface] Clean addon_a0d7b954_bitwarden application
19-11-01 19:11:09 ERROR (SyncWorker_11) [hassio.docker] Can't start addon_a0d7b954_bitwarden: 403 Client Error: Forbidden ("endpoint with name addon_a0d7b954_bitwarden already exists in network hassio")
19-11-01 19:13:25 INFO (MainThread) [hassio.updater] Fetch update data from https://version.home-assistant.io/stable.json
19-11-01 19:14:20 INFO (SyncWorker_10) [hassio.docker.interface] Clean addon_a0d7b954_bitwarden application
19-11-01 19:14:27 ERROR (SyncWorker_10) [hassio.docker] Can't start addon_a0d7b954_bitwarden: 403 Client Error: Forbidden ("endpoint with name addon_a0d7b954_bitwarden already exists in network hassio")
19-11-01 19:16:09 INFO (MainThread) [hassio.updater] Fetch update data from https://version.home-assistant.io/stable.json```

**Comment by sinclairpaul** [comment]
This actually looks like a Docker issue, can you try to run `hassio supervisor repair` from a console

**Comment by viperdream26** [comment]
@sinclairpaul Thanks so much for the suggestion!

Unfortunately, I already uninstalled Bitwarden, restarted, then restored only the Bitwarden add-on from a prior snapshot...and that fixed it. Thankfully, my hundreds of logins and everything are safe! I tried the update again and it worked this time.

Before that, I tried simply restarting (didn't work) and also restoring the 0.3.1 Bitwarden snapshot (didn't work).
